core/templates/pages/topics-and-skills-dashboard-page/skills-list/skills-list.directive.html

```diff
@@ -96,23 +101,22 @@
               </a>
             </div>
             <div class="skill-list-options-container">
-              <p class="fas fa-ellipsis-v skill-edit-box-icon protractor-test-skill-edit-box" ng-click="$ctrl.changeEditOptions(skill.id)"></p>
+              <p class="fas fa-ellipsis-v skill-edit-box-icon" ng-click="$ctrl.changeEditOptions(skill.id)"></p>
               <div ng-if="$ctrl.showEditOptions(skill.id)" class="skill-edit-options" ng-mouseleave="$ctrl.changeEditOptions(null)">
                 <div class="assign-skill-box" ng-if="getEditableTopicSummaries().length > 0 && !isUnpublishedSkill()">
                   <span class="skills-list-action-button" ng-click="$ctrl.assignSkillToTopic(skill.id)">
                     <i class="fa fa-wrench"></i>
                     <span> Assign to Topic </span>
                   </span>
                 </div>
-                <div class="merge-skill-box" ng-if="getMergeableSkillSummaries().length > 0">
-                  <div class="protractor-test-merge-skills-button"
-                       ng-click="$ctrl.mergeSkill(skill)">
+                <div class="merge-skill-box" ng-if="getMergeableSkillSummaries().length > 0" ng-click="$ctrl.mergeSkill(skill)">
```

Comment:
Should it be `$ctrl.getMergeableSkillSummaries().length`?

Reply:
Please check this for other functions used here as well.

Reply:
No these not the controller functions as such, these are being passed from the parent component.
See line 58-67 in skills-list.directive.ts

Reply:
Oh ok, thanks for explaining!